appveyor.yml

```diff
@@ -1,5 +1,5 @@
 init:
-  - git config --global core.autocrlf true
+  - git config --global core.autocrlf input
```

Comment:
`core.autocrlf=true` & `core.autocrlf=false` both change line endings in the working directory. If you try it and open up the files with Visual Studio Code, it shows you the line endings in the lower right. The setting that doesn't and the one that would be nice to have is `core.autocrlf=input`.  SourceLink will autofix the line endings by default. I think it is really slow though. I have a build queued up on AppVeyor to see just how slow. 

I searched around a bit and found this:

>you can create a build configuration executing the following command
git config --global core.longPaths true

https://confluence.jetbrains.com/display/TCD9/Git

So a solution is to create a build configuration with:
```
git config --global core.autocrlf input
```
Do you think that may work?